app/src/main/java/com/sopt/now/presentation/auth/login/LoginActivity.kt

```diff
@@ -0,0 +1,66 @@
+package com.sopt.now.presentation.auth.login
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.sopt.now.presentation.main.MainActivity
+import com.sopt.now.PreferenceUtil
+import com.sopt.now.presentation.auth.signup.SignUpActivity
+import com.sopt.now.databinding.ActivityLoginBinding
+import com.sopt.now.data.remote.dto.request.RequestLoginDto
+
+class LoginActivity : AppCompatActivity() {
+
+    private val binding by lazy { ActivityLoginBinding.inflate(layoutInflater) }
+    private val viewModel by viewModels<LoginViewModel>()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        prefs = PreferenceUtil(applicationContext)
+        initViews()
+        initObserver()
+        moveToSignUp()
+    }
+
+    private fun initViews() {
+        binding.btnLogin.setOnClickListener {
+            viewModel.login(getLoginRequestDto())
+        }
+    }
+
+    private fun initObserver() {
+        viewModel.liveData.observe(this) { loginState ->
+            Toast.makeText(
+                this@LoginActivity,
+                loginState.message,
+                Toast.LENGTH_SHORT,
+            ).show()
+            if (loginState.isSuccess) {
+                val intent = Intent(this, MainActivity::class.java)
+                startActivity(intent)
```

Comment:
```suggestion
                Intent(this, MainActivity::class.java).apply{
                    startActivity(this)
                }
```
와 같이 스코프 함수를 활용한다면, 불필요한 변수 생성을 최소화할 수 있습니다 !